clappr/src/main/kotlin/io/clappr/player/Player.kt

```diff
@@ -284,6 +296,19 @@ open class Player(private val base: BaseObject = BaseObject(),
         containerEventsIds.clear()
     }
 
+    private fun bindCoreEvents(){
+        core?.let {
+            coreEventsToListen.mapTo(coreEventsIds) { event -> listenTo(it, event) { bundle: Bundle? -> trigger(event, bundle) } }
+        }
+    }
+
+    private fun unbindCoreEvents(){
```

Comment:
Hi, what do you think about use some sintax suggar here?
```suggestion
    private fun unbindCoreEvents(){
        coreEventsIds.forEach(::stopListening)
        coreEventsIds.clear()
    }
```